public/app/plugins/panel/table/renderer.ts

```diff
@@ -292,7 +300,7 @@ export class TableRenderer {
       cellClasses.push('table-panel-cell-link');
 
       columnHtml += `
-        <a href="${cellLink}" target="${cellTarget}" data-link-tooltip data-original-title="${cellLinkTooltip}" data-placement="right"${textStyle}>
+        <a href="${cellLink}" target="${cellTarget}" data-link-tooltip data-original-title="${cellLinkTooltip}" data-placement="right"${cellStyle}>
```

Comment:
wonder if there is a potential xss attack vector here, might need to sanitize the column.style.align property of verify that it's just auto, center, left and right

Reply:
Good point @torkelo

Reply:
Yes, that makes sense, @torkelo. Thanks!
Is there a "standard" way to do it or should I just implement the check around the location you pointed?

Reply:
best way for this would be use a switch that transforms the align property into css attribute, and to not blindly add it

Reply:
What do you think of that?
@torkelo @dprokop